manifest.json

```diff
@@ -159,6 +159,26 @@
           },
           "required": ["name"]
         }
+      },
+      "taskAccessControl": {
+        "default": {},
+        "type": "object",
+        "properties": {
+          "usdPriceMax": {
+            "default": {},
+            "type": "object",
+            "properties": {
+              "collaborator": {
+                "default": null,
+                "type": "number"
+              },
+              "contributor": {
+                "default": 200,
```

Comment:
I can't decide if best practice here would be null or 0 

But I don't think we should have opinionated defaults. Instead we would just share premade configs with partners so they don't have to think about it. 

I suppose if we set this to 0 it would be easier for us to notice if there is a misconfiguration or syntax error with the yml.

Reply:
cool I've set it to 0